dataverse-webapp/src/main/resources/config/saml.properties

```diff
@@ -0,0 +1,129 @@
+#  If 'strict' is True, then the Java Toolkit will reject unsigned
```

Comment:
We need a way to externalize this properties. For example to assign certificates.

Reply:
Maybe doing it that way would be ok:
- properties would be read from file (as it is now, but not using classpath but ordinary path);
- path to properties file would be another property in `dataverse.default.properties`

?